compiler/luci/pass/src/FuseGeluPass.cpp

```diff
@@ -62,10 +89,10 @@ bool same(float a, float b) { return fabs(a - b) < 1e-5; }
  *          [Out]
  *
  */
-class GeluPattern final
+class GeluPattern final : public GeluPatternBase
 {
 public:
-  GeluPattern(luci::CircleMul *candidate)
+  GeluPattern(luci::CircleMul *candidate) : GeluPatternBase(candidate)
   {
```

Comment:
just to avoid `define but not used` error

Reply:
can you check this as the method is now removed?

Reply:
I checked, without this CI crashes with an error `define but not used`